docs/Meadow/Meadow.Foundation/Working_with_Sensors/Sensor_Service/index.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: Meadow
+title: Optimize Sensor Reads with SensorService
+subtitle: Reduce sensor thread usage by isolating reads to a the shared SensorService.
+---
+
+While developing code to read sensors asyncronously in C#, it can be very easy to create a large number of threads because of how simple async Task-based code is to write. On low-resource platforms like Meadow, too many threads can cause excessive CPU cycles as it switches between contexts. To help with this, Meadow provides the `SensorService` to help avoid this when you are regularly reading for several sensors every second or more.
```

Comment:
You can ever run out of threads.  I think we have a max of either 16 or 32 total threads (guessing it is 32)

Reply:
Added that...thank you.